acp/acp_local.go

```diff
@@ -105,6 +107,40 @@ func (l *ACPLocal) Close() error {
 	return l.manager.Terminate()
 }
 
+func (l *ACPLocal) DropAll(ctx context.Context) error {
+	err := l.Close()
+	if err != nil {
+		return err
+	}
+
+	// delete state (applicable to persistent store)
+	if l.pathToStore.HasValue() {
+		storeLocation := l.pathToStore.Value()
+		path := storeLocation + "/" + localACPStoreName
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			return fmt.Errorf("drop all: path to store does not exist")
```

Comment:
suggestion: Errors should be added to `errors.go` instead of defined inline.

Reply:
Thanks, forgot to do that. Meant to come back to it 😔

Reply:
Also rather that `drop all: xyz` error would prefer something that indicates it's an acp reset error.